script.py

```diff
@@ -0,0 +1,52 @@
+import requests
+import ipaddress
+import json
+
+
+def is_valid_ip(ip_str):
+    try:  
+        ip_obj = ipaddress.ip_address(ip_str) 
+        return True  
+    except ValueError:  
+        pass
+    return False
+
+with open('input.json') as file_object:
```

Comment:
This entire logic is redundant - simply save the config as a constant at the top of this file, not as a separate json file.